lib/services/in-memory-db.service.ts

```diff
@@ -296,6 +296,25 @@ export class InMemoryDBService<T extends InMemoryDBEntity> {
     return result$;
   }
 
+  /**
+   * Randomly generate at a set of records for the given amount and record factory.
+   * Example:
+   * ```typescript
+   * service.seed((i) => { myProp: i}, 100);
+   * ```
+   *
+   * @param recordFactory a factory method to call when generating the random record.
+   * @param amount the amount of records to generate, defaults to 10.
+   */
+  public seed(recordFactory: (index: number) => Partial<T>, amount = 10) {
+    amount = amount === null ? 10 : amount;
+
+    for (let i = 0; i < amount; i++) {
```

Comment:
Could we utilize the `createMany` method here instead of this `for` loop?

Reply:
Good point! I will switch it out.